app/build.gradle

```diff
@@ -72,6 +72,7 @@ android {
 
         debug {
             applicationIdSuffix ".debug"
+            resValue "string", "provider_authority", "de.danoeh.antennapod.debug.provider"
```

Comment:
This can also be done in `debug/res/values/strings.xml`. I don't know which method you prefer